src/datastore/postgres/schema/v14.ts

```diff
@@ -0,0 +1,12 @@
+import { IDatabase } from "pg-promise";
+
+export const runSchema = async(db: IDatabase<unknown>) => {
+    await db.none(`
```

Comment:
This seems okay, do we ever expect a case where one of these wouldn't be defined? Does this apply cleanly over existing DBs?